test/jwt/jwt_context.go

```diff
@@ -0,0 +1,27 @@
+package jwt
+
+import (
+	"context"
+
+	jwt "github.com/dgrijalva/jwt-go"
+	goajwt "github.com/goadesign/goa/middleware/security/jwt"
+)
+
+// NewJWTContext creates a context with a JWT having the given `subject`
+// using the private key that is located in the 'privateKeyPath'
+func NewJWTContext(subject, kid, privateKey string) (context.Context, error) {
```

Comment:
Not sure why you need multiple keys in your tests.. Why not just using https://github.com/fabric8-services/fabric8-common/blob/master/test/auth/auth.go#L119?

Reply:
because one of my test uses an invalid key (signed by another party) to verify that the request is not authorized

Reply:
On second thought, the token signature validation is the responsibility of the goa middleware which is invoked before the controller is called, so testing with different signing keys does not make sense at his level.